### DISCUSSION TITLE
How to style <Entry /> to not have bottom padding and bottom borderline.

### DISCUSSION THREAD

**Discussion by MadProbe** [dotnet/maui#13594] [Q&A] [answered]
I want to create a table (in Spreadsheet style) and I want to have editable text without that noise of bottom borderline and padding that goes with it.

**Comment by mhrastegari** [comment, marked as answer]
You need to use custom handlers/renderers for each platform.

https://learn.microsoft.com/en-us/dotnet/maui/user-interface/handlers/customize?view=net-maui-7.0
